src/app/app.component.ts

```diff
@@ -58,14 +60,20 @@ export class AppComponent implements OnInit, OnDestroy {
     this.subscriptionList$.push(
       this.store.select(getAuthStatus).subscribe((data: boolean) => {
         if (data) {
-          this.orderSub$ = this.checkoutService.fetchCurrentOrder().subscribe();
+          this.orderSub$ = this.checkoutService.fetchCurrentOrder()
+            .subscribe(_ => { },
+              _ => {
+                localStorage.clear()
+                this.store.dispatch(this.authAction.logoutSuccess())
```

Comment:
Why are you dispatching logout success on init?

Reply:
It is to change the authstate in store. If token is expire.